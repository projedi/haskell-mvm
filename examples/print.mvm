print(1);
print(2.3);
print("abc\n");
print(1e9, " ", 0.3e-5, "\n");
print("Variadic", 100, -0.01);
