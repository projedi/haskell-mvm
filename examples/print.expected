12.3abc
1e+09 3e-06
Variadic100-0.01
